Add optimization tips for mobile in Exporting to HTML5

Calinou opened the pull request:
See https://github.com/godotengine/godot/issues/60914.

cc @mossfrog

clayjohn commented:
We should also mention the problem with importing as sRGB in iOS 13 and below Users will need to disable sRGB and "detect 3D" as per https://github.com/godotengine/godot/issues/60914#issuecomment-1127237932

mhilbrunner commented:
cc @Faless just in case you have anything non-obvious to add :)

mhilbrunner reviewed (commented):
Fabio seems busy, @Calinou if you add the above, I'm happy with merging as is. Looks good to me!

Calinou commented:
> Fabio seems busy, @Calinou if you add the above, I'm happy with merging as is. Looks good to me!

I still need to mention the sRGB workaround for iOS < 13. Note that disabling Detect 3D will also disable mipmap generation, so it'll have to be explicitly enabled by the user to prevent textures from looking grainy in 3D.

mhilbrunner commented:
@Calinou Friendly bump. :)

Calinou commented:
Rebased and updated for 4.3.stable. The iOS 13/15 advice is no longer relevant nowadays, so I removed it. It was most likely fixed on Godot or Apple's side with the new Compatibility renderer in 4.0, and iOS 13 is EOL now.

skyace65 commented:
Thanks!
